### DISCUSSION TITLE
Does MicroPython accept small donations?

### DISCUSSION THREAD

**Discussion by liudr** [micropython/micropython#16464] [General]
End of the year, expressing gratitude, not intending to influence the direction of the project type of small donation?

**Comment by andrewleech** [comment]
GitHub sponsors is the main way donations are welcomed. It looks like one-off donations are available there too!

The Sponsor button is on the main page of the micropython GitHub or direct link here: https://github.com/sponsors/micropython

**Reply by liudr** [reply]
Thanks! I managed to become a one-time sponsor. Will do it again next year etc.
